### DISCUSSION TITLE
Deployment not triggered by commits

### DISCUSSION THREAD

**Discussion by lucabertolasi** [vercel/community#6105] [Account Issue] [answered]
### Question

Hi all,
Could you please help? (It was working yesterday night)

Problem: Deployments not triggering on commit push

Problem solving so far:
- I've already read https://vercel.com/guides/why-aren-t-commits-triggering-deployments-on-vercel and my git config is OK
- Github app connected OK

So odd, it worked like a charm 2 days ago.

Thanks in advance - Best,
L.

**Comment by ryanoliverdev** [comment]
Also experiencing this, I deployed 3 hours ago and it triggered automatically. Probably an issue that just happened.

**Reply by homanp** [reply]
Same here. Something fishy going on.

**Reply by mattnicholson** [reply]
Same - manual deploys are building fine, but auto deployments aren't triggering from git

**Reply by KevinSig** [reply]
Same here, but [status](https://www.vercel-status.com/) shows just fine ..

**Reply by jrhizor** [reply]
Same here, it's been working for months for me and stopped today.

**Reply by CyanideData2** [reply]
Same for me

**Reply by mattnicholson** [reply]
Looks like it's working again now — must have been a really long queue as I can see the deployments that should have happened yesterday have now pulled through

**Comment by amyegan** [comment, marked as answer]
Thanks for reporting this. There was a deployment incident, but things should be working as normal now.

https://www.vercel-status.com/incidents/4ngn25s3jkgr

**Comment by lucabertolasi** [comment]
Thank you for sorting this! best
